e2e/lib/config.go

```diff
@@ -41,6 +41,8 @@ type Config struct {
 	NodePrivKeyPathList     []string
 	NodeRPCAddressList      []string
 	NodeProxyAppAddressList []string
+
+	AlwasyApphashCheck      bool
```

Comment:
This is supposed to control whether or not the app hash check happens at the end of the test or not, right? Then `CheckAppHashOnExit` might be a better name.